rest/taskana-rest-spring-example-wildfly/src/main/resources/application.properties

```diff
@@ -1,41 +1,44 @@
-spring.profiles.active=@activatedProperties@
```

Comment:
Please run `git config --global core.autocrlf input` so that the line endings are unix and not windows :)